### ISSUE TITLE
docz-theme-default@latest incompatible with simple-props-table

### ISSUE THREAD

**Issue by 100ideas** [dfee/rbx#60]
Wanted to run a local dev version to play around in docz. Cloned, npm installed, tried running `npm run docs`...

and make complained about not having a target: There is a mismatch between how the make tasks are named in the Makefile and how they are called from npm package.json scripts. Makefile uses hyphen like 'docs-dev' but package.json uses underscore like 'docs_dev'. 

So fixed that, tried again, and found typo in `src/components/navbar/__docs__/navbar.docs.mdx`. Fixed that (#59), tried again...

... but  now`docz dev` is complaining about about itself:

```
./node_modules/docz-theme-default/node_modules/docz/dist/index.m.js
Module not found: Can't resolve '~db' in './rbx/node_modules/docz-theme-default/node_modules/docz/dist'
```

so upgraded `docz-theme-default` to  `latest` (v1.0.4) (#57), deleted `./docz/` cache, and ran `$ ./node_modules/.bin/docz dev`... and got react errors in browser when looking at rbx components in docz:

```
(BaseSimplePropsTable, in code (at simple-props-table.tsx:131)) Invariant Violation: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports. Check the render method of `BaseSimplePropsTable`.
```

The components do render and can be clicked. there's just a big error box. 😢 

![rbx-issue-54](https://user-images.githubusercontent.com/57006/56421618-3fe30f00-6258-11e9-874d-e27497b71379.png)

Lastly, in package.json, should build command be 'make' or 'make build'? latter doesn't work for me.

---

in conclusion:
1. package.json scripts calling make commands don't work
2. docz without latest theme throws error about not finding `~db`
3. docz with latest theme (#57) has problems with `simple-props-table.tsx`
   - however, this component does not seem necessary for the docz site to mostly work?

possible solution: remove `simple-props-table.tsx` for now.

**Comment by dfee** [comment]
> 
    1. package.json scripts calling make commands don't work

updated. must've changed it in the makefile w/o updating it in package.json

    2. docz without latest theme throws error about not finding ~db

there are quite a few errors now that i've updated all packages. will try and get this up and running shortly.

    3. docz with latest theme (#57) has problems with simple-props-table.tsx
        • however, this component does not seem necessary for the docz site to mostly work?

same as with #2.

**Comment by dfee** [comment]
Fixed in `next` branch, which will soon be merged into master (with a new version pushed).
